### ISSUE TITLE
Selecting Display to Show wvkbd on

### ISSUE THREAD

**Issue by snyball** [jjsullivan5196/wvkbd#38]
Hi! This project is really useful for using a laptop connected to a TV with only a mouse, but I don't see a way to select the display that wvkbd is shown on. I suggest adding this as a new command-line option.

**Comment by proycon** [comment]
I agree. Though the command-line option is not implemented yet, the above patch by @mojyack ensures it starts starts on the current display rather than always on the first one.

**Comment by niteria** [comment]
I had clankers add it for me, maybe it would be useful to someone: https://github.com/niteria/wvkbd/commit/76cc9460c709e95a762f22da9750fe68a1d9ad46
